src/charm.py

```diff
@@ -98,6 +102,10 @@ def __init__(self, *args: Any) -> None:
         self._db_relation_name = "pg-database"
         self._hydra_relation_name = "endpoint-info"
         self._login_ui_relation_name = "ui-endpoint-info"
+        self._prometheus_scrape_relation_name = "metrics-endpoint"
+        self._loki_push_api_relation_name = "logging"
+        self._kratos_service_command = "kratos serve all"
```

Comment:
Why move this to a variable?

Reply:
relation names are variables, also with loki there's a confusion about what the default relation name is. Also I assumed it's better to use a variable string then to hardcode the command into the pebble layer.

Reply:
also this way the charm code is more uniform across the bundle

Reply:
What's the confusion about the default loki relation name about? Isn't it just [logging](https://github.com/canonical/kratos-operator/blob/9835be701fccf3ef8454f993b89f7729da09eac9/lib/charms/loki_k8s/v0/loki_push_api.py#L491)?